lib/tasks/update_lineagedb.rake

```diff
@@ -4,14 +4,30 @@ require 'English'
 
 desc 'Imports NCBI lineage data into IDseq'
 
-task 'update_lineage_db', [:dryrun] => :environment do |_t, args|
+# Lineage records have a [version_start, version_end] inclusive range. Each
+# pipline run has an AlignmentConfig that has a lineage_version. The pipeline
```

Comment:
To be slightly more precise, I would say something like: "The version of the NCBI taxonomy that specifies the lineages is kept in sync with the version of NCBI NT/NR used during alignment by comparing alignment_config.lineage_version to taxon_lineages.version_start / taxon_lineages.version_end. When serving reports, the correct lineage should be selected using the following condition: ..."
(you'll definitely word it better than me)

Reply:
I added some more docs but I'm still not sure if I captured what you wanted. 

> When serving reports, the correct lineage should be selected 

From my reading of the code, the correct lineage is determined at write-time in `generate_aggregate_counts`. The report then merely selects the records associated with the current pipeline run.

Reply:
It's found dynamically when showing the taxon tree: https://github.com/chanzuckerberg/idseq-web/blob/e114826b65f9029af2e60257017dbbbb54f90c32/app/models/taxon_lineage.rb#L196